kombu/transport/redis.py

```diff
@@ -1064,8 +1069,12 @@ def on_poll_start():
             [add_reader(fd, on_readable, fd) for fd in cycle.fds]
         loop.on_tick.add(on_poll_start)
         loop.call_repeatedly(10, cycle.maybe_restore_messages)
+        health_check_interval = connection.client.transport_options.get(
```

Comment:
@georgepsarakis I don't feel great about this. My goal here is to get the passed in `health_check_interval` transport option. I think this will do it, but if someone has a better way, I would happily change it.

Reply:
How about adding it to the `Channel` instead as a method (since a property already exists)?

```python
health_check_interval = self.channels[0].get_health_check_interval()
```

I am not sure if this is the right interface either since `Transport` uses a list of channels, but I think it is better.